lib/index.js

```diff
@@ -40,6 +77,30 @@ module.exports = class {
       `${this.base}node.${this.format}?${qs.stringify(filterOptions, { indices: false })}`);
   }
   /**
+   * Node Comment Id
+   * @param  {Object} [filterOptions={}]
+   *   Query string filter options
+   * @return {Promise}
+   *   Pending issueRequest promise.
+   */
+  nodeCommentId(filterOptions = {}) {
+    return this.cidRequest(
+      this.methods.GET,
+      `${this.base}node.${this.format}?${qs.stringify(filterOptions, { indices: false })}`);
+  }
+  /**
+   * Node Taxonomy Id
+   * @param  {Object} [filterOptions={}]
+   *   Query string filter options
+   * @return {Promise}
+   *   Pending issueRequest promise.
+   */
+  nodeTaxonomyId(filterOptions = {}) {
```

Comment:
Lets normalize the method name, perhaps something like `getRelatedTaxonomyTerms`?